draft-ietf-quic-recovery.md

```diff
@@ -632,8 +632,9 @@ in most cases and is less likely to spuriously retransmit data.
 
 Resumed connections over the same network MAY use the previous connection's
 final smoothed RTT value as the resumed connection's initial RTT.  When no
-previous RTT is available, the initial RTT SHOULD be set to 333ms, resulting in
-a 1 second initial timeout, as recommended in {{?RFC6298}}.
+previous RTT is available, the initial RTT SHOULD be set to 333ms.  This
```

Comment:
I prefer the existing text to the updated text.  Is there something you're adding that is important?

Reply:
"initial timeout" seems to be a concept that doesn't exist in 6298, but it seems to be implied something.  And it is not defined.  So I wanted to remove it.

Reply:
Oh, I forgot... The PTO is 3*kInitialRtt only for the handshake.  A new path will be after the handshake is complete, so the value will have max_ack_delay added.